lib/plugins/spawn/compileServiceCode/java/index.js

```diff
@@ -0,0 +1,30 @@
+var tmp = require('tmp');
+var path = require('path');
+var fs = require("fs");
+var spawnSync = require('child_process').spawnSync;
+
+function getClassNameFromSource(code) {
```

Comment:
Yeah I really like this (by looking at the code). I'll try and run / test this (if possible already, by the looks of it I assume it is :))